### ISSUE TITLE
ERROR ~ Unknown config attribute: igenome_base

### ISSUE THREAD

**Issue by mhebrard** [SciLifeLab/NGI-RNAseq#233]
Hi, 
I try to use NGI-RNAseq for the first time, so I might forget something.

I am on windows and installed Singularity using vagrant
I use vagrant to launch a linux machine with VirtualBox (**Ubuntu 16.04**)
in this machine, I installed **Singularity 2.4** and **Nextflow 0.29.1**
Then I run:

```
nextflow run SciLifeLab/NGI-RNAseq -with-singularity --genome GRCm38 --reads '/vagrant/Data/*_R{1,2}*.fastq.gz'
```

I get the trace
```
N E X T F L O W ~ version 0.29.1
Pulling SciLifeLab/NGI-RNAseq ...
 downloaded from https://github.com/SciLifeLab/NGI-RNAseq.git
Launching `SciLifeLab/NGI-RNAseq` [elegant_davinci] - revision: a21a1077ec [master]
ERROR ~ Unknown config attribute: igenome_base -- check config file: 
/home/vagrant/.nextflow/assets/SciLifeLab/NGI-RNAseq/nextflow.config

null

 -- Check '.nextflow.log' file for details
```

It looks like an issue with the variables that compose the URL to download the reference genome... do you have any advice ?

**Comment by ewels** [comment]
Hi @mhebrard,

Thanks for this - it's a known bug. We've actually moved this pipeline over to the new nf-core community here: https://github.com/nf-core/rnaseq (see http://nf-co.re). I fixed it in that version of the pipeline but haven't applied the patch here.

You're not the first to mention this, so I should really do a minor release here just fixing this bug. Then I should archive this pipeline and redirect to the new nf-core one.

Now I just need to remember _how_ I fixed it..!

Phil
